packages/ods/src/components/timepicker/src/constant/timezone-preset.ts

```diff
@@ -2,12 +2,16 @@ enum ODS_TIMEZONES_PRESET {
   all = 'all'
 }
 
+/** @deprecated use `OdsTimezonesPreset` instead */
```

Comment:
if we can't get it on the doc generation, we should at least manually add a warn message on the Technical page of the component